website/docs/docs/integrate/quickstart-semantic-layer.md

```diff
@@ -0,0 +1,292 @@
+    time_grains: [day, week, month, year]
+
+    dimensions:
+      - customer_status
+      - had_credit_card_payment
+      - had_coupon_payment
+      - had_bank_transfer_payment
+      - had_gift_card_payment
+
+    filters:
+      - field: status
+        operator: '='
+        value: "'completed'"
+```
+</VersionBlock>       
+
+1. Click **Save** and **Compile** the code.
+2. Commit and merge the code changes that contain the metric definitions.
+3. If you'd like to further design and define your own metrics, review the following documentation:
+
+    - [dbt metrics](/docs/building-a-dbt-project/metrics) will povide you in-depth detail on attributes, properties, filters, and how to define and query metrics.
+   
+    - Visit the [Structure and design your metric definition](URL) page to understand best practices for designing and structuring metrics in your dbt project.
+
+## Develop and query metrics
+
+You can dynamically develop and query metrics directly in dbt and verify their accuracy *before* running a job in the deployment environment by using the `metrics.calculate` and `metrics.develop` macros.
+
+To understand when and how to use the macros above, review [dbt metrics](/docs/building-a-dbt-project/metrics) and make sure you install the [dbt_metrics package](https://github.com/dbt-labs/dbt_metrics) first before using the above macros.
+
+:::info 📌 
+
+**Note:** you will need access to dbt Cloud and the dbt Semantic Layer from your integrated tool of choice. 
+
+:::
+
+## Run your production job
+
+Once you’ve defined metrics in your dbt project, you can perform a job run in your deployment environment to materialize your metrics. The deployment environment is only supported for the dbt Semantic Layer at this moment. 
+
+1. Go to **Deploy** in the navigation and select **Jobs** to re-run the job with the most recent code in the deployment environment.
+2. Your metric should appear as red nodes in the dbt Cloud IDE and dbt directed acyclic graphs (DAG).
+
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/metrics_red_nodes.png" title="DAG with metrics appearing as red nodes" />
+
+
+**What’s happening internally?**
+
+- Merging the code into your main branch allows dbt Cloud to pull those changes and builds the definition in the manifest produced by the run.
+- Re-running the job in the deployment environment helps materialize the models, which the metrics depend on, in the data platform. It also makes sure that the manifest is up to date.
+- Your dbt Metadata API pulls in the most recent manifest and allows your integration information to extract metadata from it.
+
+## Set up dbt Semantic Layer
+
+Before continuing, you must have a multi-tenant dbt Cloud Team or Enterprise plan. Developer accounts will be able to query the Proxy Server using SQL but will not be able to browse dbt metrics in external tools, which requires access to the [Metadata API](/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview).
```

Comment:
Same comment as above. This should just say "multi-tenant dbt Cloud plan", because the following sentence then says that Developer plans can also use the Semantic Layer, just with a couple of functionality limitations.